### ISSUE TITLE
Temp, Humidity reading incorrect

### ISSUE THREAD

**Issue by A-vesalius** [Luligu/matterbridge-eve-weather#1]
Have eve weather in Homekit Temp is correctly listed as 89 °F and Humidity 66% 

The device in matterbridge-eve-weather list this as Temperature: 18.51°C (65.10 °F) Humidity: 42.71%

**Comment by Luligu** [comment]
Hi,
can you explain ?

**Comment by A-vesalius** [comment]
Well over the weekend I updated the Eve weather to the newly released Native Matter firmware from eve, so I won't need the Matterbridge for the eve weather any longer as I can now natively pair with Apple Home and Home Assistant at the same time.

But I had the Eve weather paired with Homekit over thread for a year or so before trying this plugin. For a week I used your plugin to also pair the Eve weather into Home Assistant while leaving the device paired natively to Apple Home. The temp/humidity on the Eve weather display and in homekit was the same; however, the Temp/Humidity listed in the MatterBridge logs and Home assistant for the Eve weather was incorrect and stayed incorrect for the duration of my use (numbers listed above).
